app/models/enhancements/artefact.rb

```diff
@@ -8,6 +8,11 @@ class Artefact
   # without persisting it
   attr_accessor :indexable_content
 
+  # Add a non-field attribute so we can pass the redirect_url over to the Router
+  # without persisting it
+  attr_accessor :redirect_url
```

Comment:
Ah, ok, that makes sense. Do I just need to store it, or is there a "re-register all the things" task I need to update?